src/components/Property/RepairsHistory/RepairsHistoryTable.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types'
+import RepairsHistoryRow from './RepairsHistoryRow'
+
+const RepairsHistoryTable = ({ workOrders }) => (
+  <table className="govuk-table govuk-!-margin-top-5 repairs-history-table">
+    <thead className="govuk-table__head">
+      <tr className="govuk-table__row govuk-body">
+        <th scope="col" className="govuk-table__header">
+          Reference
+        </th>
+        <th scope="col" className="govuk-table__header">
+          Date raised
+        </th>
+        <th scope="col" className="govuk-table__header">
+          Status
+        </th>
+        <th scope="col" className="govuk-table__header">
+          Description
+        </th>
+      </tr>
+    </thead>
+    <tbody className="govuk-table__body">
+      {workOrders.map((workOrder, index) => (
+        <RepairsHistoryRow key={index} {...workOrder} />
+      ))}
+    </tbody>
+  </table>
+)
+
+RepairsHistoryTable.propTypes = {
+  workOrders: PropTypes.arrayOf(
+    PropTypes.shape({
+      reference: PropTypes.number,
+      dateRaised: PropTypes.instanceOf(Date),
+      description: PropTypes.string,
```

Comment:
Do we want `status: PropTypes.string` here, too? To match proptype [here](https://github.com/LBHackney-IT/repairs-hub-frontend/pull/46/files?file-filters%5B%5D=.js&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-db60a5e567c53120c92b6d3c0238f4639c9de128670857135cd4efb1b308f63cR37)

Reply:
Yeah, good spot thanks